core/src/Kokkos_View.hpp

```diff
@@ -821,21 +821,11 @@ class View : public ViewTraits<DataType, Properties...> {
     };
   };
 
```

Comment:
Goes with my other comment: this change increases the number of function inlines the compiler has to do for every data access. I don't really want that.

Reply:
Let me see if the warning still appears and if I find a different solution.

Reply:
I found a different solution.